doc/config-schemas/setAddAlarmUser.md

```diff
@@ -0,0 +1,46 @@
+# Set Add user To Alarm partition
+
+## Descripcion
+
+Agrega un usuario a una particion de alarma
+
+## Request Message
+
+```json
+{
+    ...
+    "data": {
+        "deviceData": {...},
+        "configKey": "setAddAlarmUser",
+        "value":{
+            "systemId":"xxx-xxxx",
+            "userName": "Em-32",
+            "accessCode": "5125"
+        }
+    }
+}
+
+```
+
+| Campo      | Tipo   | Descripcion                   |
+| ---------- | ------ | ----------------------------- |
+| systemId   | string | Id de sistema de la particion |
+| userName   | string | Nombre del usuario            |
+| accessCode | string | codigo de acceso              |
+
```

Comment:
Why not here need the `number` propiertie in the request value, for example in `setAlarmPartition` the `number` is used in the value, why not here?